modules/fedimint-ln-client/src/pay.rs

```diff
@@ -643,13 +643,7 @@ impl PaymentData {
     }
 
     pub fn is_expired(&self) -> bool {
-        let Some(now) = now().duration_since(UNIX_EPOCH).map(|t| t.as_secs()).ok() else {
-            // Something is very wrong (time out of bounds), we'll not attempt too pay the
-            // invoice
-            return true;
-        };
-
-        self.expiry_timestamp() < now
+        self.expiry_timestamp() < duration_since_epoch().as_secs()
```

Comment:
**thought:** Not a problem of this PR, but would rename `expiry_timestamp` to `expiry_timestamp_secs` or something.